src/validations/html/slowparse.js

```diff
@@ -66,6 +71,7 @@ const errorMap = {
   UNCLOSED_TAG: error => ({
     reason: 'unclosed-tag',
     payload: {tag: error.openTag.name},
+
```

Comment:
fyi @outoftime will freak out about this

Reply:
removed